clang/test/Analysis/Checkers/WebKit/uncounted-lambda-captures.cpp

```diff
@@ -372,6 +372,14 @@ void trivial_lambda() {
   trivial_lambda();
 }
 
+bool call_lambda_var_decl() {
+  RefCountable* ref_countable = make_obj();
+  auto lambda = [&]() -> bool {
+    return ref_countable->next();
+  };
+  return lambda();
+}
+
```

Comment:
Maybe add a test for capture by copy also? (for completeness)

Reply:
Sure, added.